index.bs

```diff
@@ -39,8 +39,9 @@ Level: none
 Repository: w3c/process
 Shortname: process
 Abstract:
-	The mission of the World Wide Web Consortium (<abbr>W3C</abbr>) is to lead the World Wide Web to its full potential
-	by developing common protocols that promote its evolution and ensure its interoperability.
+	The <a href="https://www.w3.org/mission/">mission of the World Wide Web Consortium (W3C)</a>
+        is making the web work, for everyone. 
+        W3C brings together global stakeholders to develop open standards which enable a world wide web that connects and empowers humanity.
```

Comment:
```suggestion
        W3C brings together global stakeholders to develop open standards that enable a world wide web that connects and empowers humanity.
```

Matches the recently updated text on https://www.w3.org/mission/, as per the discussion in https://github.com/w3c/process/issues/989#issuecomment-2682033481